src/derivers/bip39.ts

```diff
@@ -79,3 +85,33 @@ export async function createBip39KeyFromSeed(
     curve: curve.name,
   });
 }
+
+/**
+ * Create a {@link SLIP10Node} from BIP-39 entropy.
+ *
+ * @param entropy - The BIP-39 entropy.
+ * @param curve - The curve to use.
+ * @returns An object containing the corresponding BIP-39 master key and chain code.
+ */
+export async function createBip39KeyFromEntropy(
```

Comment:
```suggestion
export async function createBip39Cip3IcarusKeyFromEntropy(
```
In line with https://github.com/nufi-official/metamask-key-tree/pull/1/files#r1410675414 I'd consider adding "Cip3/Icarus" explicitly in the name of the function to remove any confusion and draw a clear difference from `createBip39KeyFromSeed`